scrapper.py

```diff
@@ -1,7 +1,8 @@
 """
 Scrapper implementation
 """
-
+import os, json, requests
```

Comment:
it is a better practice do this kind of imports separately
plus, they should be divided into three blocks (built-in, external and local)  and sorted alphabetically